java/src/main/java/com/google/cloud/dataproc/templates/hbase/README.md

```diff
@@ -0,0 +1,49 @@
+## 1. Hive To BigQuery
+### Required JAR files
+
+Some HBase dependencies are required to be passed when submitting the job.  
+These dependencies need to be passed by using the --jars flag, or, in the case of Dataproc Templates, using the JARS environment variable.  
+Some dependencies (jars) must be downloaded from [MVN Repository](https://mvnrepository.com/) and stored your GCS bucket (create one to store the dependencies).
+
+- **[Apache HBase Spark Connector](https://mvnrepository.com/artifact/org.apache.hbase.connectors.spark/hbase-spark) dependencies (already mounted in Dataproc Serverless, so you refer to them using file://):**
```

Comment:
I tried to include these jars along with pom.xml, but got ClassNotFoundException, one mentioned in the mail thread for HbaseToGCS python template, hence, mostly due to compilation of code in different environment. This may be encountered by others as well, so thought of passing it along with other JARS to avoid such conflicts and errors.